physical_properties/pKa/experimental_data/README.md

```diff
@@ -8,7 +8,11 @@ Since pKa values measured by Sirius T3 were reported with 2 decimal places, we h
 
 ## Manifest
 - `pKa_experimental_values.csv` - Mean and SEM of pKa values organized in a table. Up to three pKa values were reported based on how many pKas were measured in the experimental interval of 2-12.
-- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values.
-- `pKa_results_of_replicate_experiments.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory).
+- `calc_pKa_value_statistics.py` - Python script used to calculate mean and SEM of pKa values from `pKa_results_of_replicate_experiments.csv`.
+- `pKa_results_of_replicate_experiments.csv` - Experimental results of independent replicates of pKa measurements(N=3) and names of experimental reports (PDF files can be found in `experiment_reports\` directory). For molecules with pKas measured both in 100% water and with cosolvent, pKa measurements done in 100% water are selected to be used for SAMPL6 challenge analysis considering potentially higher accuracy of pKa measurements in 100% water.

```

Comment:
"pKa measurements done in 100% water are selected to be used for SAMPL6 challenge analysis considering potentially higher accuracy of pKa measurements in 100% water."

In this section you don't need both "in 100% water"s

Reply:
Corrected.